tide_core.module

```diff
@@ -189,3 +189,16 @@ function tide_core_entity_bundle_create($entity_type_id, $bundle) {
     }
   }
 }
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ */
+function vicgovau_core_form_content_moderation_entity_moderation_form_alter(&$form, FormStateInterface $form_state, $form_id) {
```

Comment:
@vincent-gao should this be `tide_core_form_content_moderation_entity_moderation_form_alter`?

Reply:
@anthony-malkoun changed. thank you!